source/common/res/features/shared/main.js

```diff
@@ -325,6 +325,38 @@ ynabToolKit.shared = (function () {
       this.showModal(title, message, 'close');
     },
 
+    showNewReleaseModal(version) {
+      let $modal = $(`<div class="toolkit-modal-overlay">
+                        <div class="toolkit-modal">
+                          <div class="toolkit-modal-header">
+                            <img src="https://camo.githubusercontent.com/d57a1d819550e8ccbfdfa2552df2cfb5c5ec28e7/687474703a2f2f692e696d6775722e636f6d2f534a68774270552e706e67" />
+                          </div>
+                          <div class="toolkit-modal-content">
+                            <h1>The Toolkit for YNAB extension has been updated!</h1>
+                            <span class="version">You are now using version ${version}. View the <a href="https://forum.youneedabudget.com/categories/ynab-extensions" target="_blank">release notes</a>.</span>
```

Comment:
Just so you know, the use of a template string here might theoretically get us rejected from Firefox review because they see it as a vector for XSS, but given that we control that version string ourselves I think it'll be ok. Just more an informational comment to let you know why most of the toolkit uses the `$('<div>', ...)` style for constructing DOM.

Reply:
Ah I see. I was following the previous showModal functions pattern with backticks. So should multi-line strings be constructed like this:
let $modal = $('<div class="toolkit-modal-overlay">') +
                        $('<div class="toolkit-modal">') +
Or am I misunderstanding?